### PR TITLE
certgraph 20180911 (new formula)

### PR DESCRIPTION
**Pull request by betrisey**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
New formula for CertGraph: https://github.com/lanrat/certgraph

### CODE REVIEW

File: Formula/certgraph.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Certgraph < Formula
+  desc "Intelligence tool to crawl the graph of certificate Alternate Names"
+  homepage "https://lanrat.github.io/certgraph"
+  url "https://github.com/lanrat/certgraph.git",
+    :tag => "20180217",
```
Comment (DomT4) [inline Formula/certgraph.rb]: 4 space indent rather than 2 space on both `:tag` and `:revision` (but nowhere else).

File: Formula/certgraph.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/lanrat/certgraph"
+    dir.install Dir["*"]
+
+    cd dir do
+      system "make", "dep"
+      system "make"
+      bin.install "certgraph"
```
Comment (DomT4) [inline Formula/certgraph.rb]: You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

File: Formula/certgraph.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    end
+  end
+
+  test do
+    system "#{bin}/certgraph", "-version"
```
Comment (DomT4) [inline Formula/certgraph.rb]: Any chance of a test that exercises a deeper level of functionality? Thanks!

**Review by DomT4 — CHANGES_REQUESTED** [review]

**Comment by betrisey** [comment]
Thank you for the feedback! I made the changes.

**Comment by DomT4** [comment]
We're going to need upstream to pin this dependency to a commit or a tag before we can accept this:
```
Root project is "github.com/lanrat/certgraph"
 9 transitively valid internal packages
 1 external packages imported from 1 projects
(0)   ✓ select (root)
(1)	? attempt github.com/lib/pq with 1 pkgs; 3 versions to try
(1)	    try github.com/lib/pq@v1.0.0
(2)	✗   github.com/lib/pq@v1.0.0 not allowed by constraint master:
(2)	    master from (root)
(1)	    try github.com/lib/pq@master
(1)	✓ select github.com/lib/pq@master w/2 pkgs
  ✓ found solution with 2 packages from 1 projects
```

**Comment by betrisey** [comment]
Hi @DomT4 
The author of cergraph released a new version. It is now using Go modules and the dependencies are pinned.

**Comment by betrisey** [comment]
Hi @fxcoudert 
I saw that the old reviewer resigned today (https://github.com/Homebrew/brew/commit/da8d37ee31d63554ca6f14261c22fa57f0331c66).
Could you review it please?
Thank you

**Pull request closed without merging**